tools/relay/README.md

```diff
@@ -0,0 +1,47 @@
+# Relay
+
+Relay will act as an interface between invoker and the benchmark server, allowing the user of a function to supply a custom form of input than a strict Helloworld format (see `aes.proto` for example).
```

Comment:
Maybe write .. allowing users to add function benchmarks with any arbitrary protocol and not been constrained to the helloworld protocol the invoker use to trigger requests.

However, @ustiugov might be a better expert in how to explain it.

Reply:
Please check. Later when the complete functionality is done we should also add a doc on how to add clients for relay.